engine/src/main/java/com/datatorrent/stram/util/SecurityUtils.java

```diff
@@ -87,9 +105,59 @@ public static boolean isHadoopWebSecurityEnabled()
     return hadoopWebSecurityEnabled;
   }
 
-  public static boolean isStramWebSecurityEnabled()
+  @VisibleForTesting
+  protected static boolean isStramWebSecurityEnabled()
   {
     return stramWebSecurityEnabled;
   }
 
+  /**
+   * Setup security related configuration for {@link org.apache.hadoop.yarn.webapp.WebApp}.
+   * @param config
+   * @param sslConfig
+   * @return
+   */
+  public static Configuration configureWebAppSecurity(Configuration config, SSLConfig sslConfig)
```

Comment:
How about calling this method get instead of configure as configure sounds like it is configuring something whereas the method is returning the necessary configuration which is being used later to setup the web service.